api/libproject/generate_appengine_web_xml.sh

```diff
@@ -1,6 +1,22 @@
 #!/bin/bash
-set -euo pipefail
```

Comment:
do we want to keep this command now that you aren't debugging anymore?  probably want this script to fail on error

Reply:
added